README.md

```diff
@@ -85,40 +85,41 @@ data can be retrieved through either the `data` event or by piping the instance
-| [options.useAutoCheckpoints]        | <code>boolean</code>                        | <code>true</code>             | Set to `true` to make the client automatically store shard checkpoints using the sequence number of the most-recently received record. If set to `false` consumers can use the `setCheckpoint()` function to store any sequence number as the checkpoint for the shard.                   |
-| [options.useAutoShardAssignment]    | <code>boolean</code>                        | <code>true</code>             | Set to `true` to automatically assign the stream shards to the active consumers in the same group (so only one client reads from one shard at the same time). Set to `false` to make the client read from all shards.                                                                     |
-| [options.useEnhancedFanOut]         | <code>boolean</code>                        | <code>false</code>            | Set to `true` to make the client use enhanced fan-out consumers to read from shards.                                                                                                                                                                                                      |
-| [options.usePausedPolling]          | <code>boolean</code>                        | <code>false</code>            | Set to `true` to make the client not to poll for more records until the consumer calls `continuePolling()`. This option is useful when consumers want to make sure the records are fully processed before receiving more (only applicable when `useEnhancedFanOut` is set to `false`)     |
-| [options.useS3ForLargeItems]        | <code>boolean</code>                        | <code>false</code>            | Whether to automatically use an S3 bucket to store large items or not.                                                                                                                                                                                                                    |
-| [options.leaseAcquisitionInterval]  | <code>number</code>                         | <code>20000</code>            | The interval in milliseconds for how often to attempt lease acquisitions.                                                                                                                                                                                                                 |
+| Param                                      | Type                                        | Default                       | Description                                                                                                                                                                                                                                                                               |
```

Comment:
What's this whole change?

Reply:
It appears to be an automated update for the readme file.
I didn't modify the readme, but I can see where it took comments I added and added them here.